packages/wonder-blocks-clickable/src/components/clickable-behavior.js

```diff
@@ -223,7 +223,9 @@ const disabledHandlers = {
     onKeyUp: () => void 0,
     onFocus: () => void 0,
     onBlur: () => void 0,
-    tabIndex: -1,
+    // Clickable components should still be tabbable so they can
+    // be used as anchors.
+    tabIndex: 0,
 };
```

Comment:
I think it would be good to have some unit tests that verify that this is the case.

Reply:
Nisha and I worked on implementing some focusable tests - but testing focus in RTL (or even Enzyme) is a giant mess. And in the end we're not even confident that it's working correctly.